components/messages/HowTo.module.css

```diff
@@ -0,0 +1,27 @@
+.howtocontainer {
+  height: 100%;
+  display: grid;
+  grid-template-rows: repeat(6, 1fr);
+  background-color: var(--main-color);
+}
```

Comment:
You could add `align-items: center` to the container and remove lines 13 and 22 (align-self)